tests/execution/graphql-basic-query.md

```diff
@@ -0,0 +1,51 @@
+```graphql @config
+schema @server(port: 8000) @upstream(baseURL: "http://upstream/graphql") {
+  query: Query
+}
+
+type Query {
+  posts: PostsConnection @graphQL(name: "allPosts")
+}
+
+type Post implements Node {
+  id: ID!
+  title: String!
+  body: String!
+}
+
+type PostsConnection {
+  posts: [Post]
+  totalCount: Int
+}
+
+interface Node {
+  id: ID!
+}
+```
+
+```yml @mock
```

Comment:
Tried to test this PR by reverting this change but no tests failed.

Reply:
also we have couple of tests for graphql datasource, could you check them?